user-operator/src/main/java/io/strimzi/operator/user/UserOperatorConfig.java

```diff
@@ -115,6 +123,25 @@ public static UserOperatorConfig fromMap(Map<String, String> map) {
         return new UserOperatorConfig(namespace, reconciliationInterval, zookeeperConnect, zookeeperSessionTimeoutMs, labels, caCertSecretName, caKeySecretName, caNamespace);
     }
 
+    public static int getClientsCaValidityDays() {
+        return getIntProperty(UserOperatorConfig.STRIMZI_CLIENTS_CA_VALIDITY);
+    }
+
+    public static int getClientsCaRenewalDays() {
+        return getIntProperty(UserOperatorConfig.STRIMZI_CLIENTS_CA_RENEWAL);
+    }
+
+    private static int getIntProperty(String name) {
+        // default value is set in operator
+        Map<String, String> map = System.getenv();
+        int value = 0;
+        String intEnvVar = map.get(name);
+        if (intEnvVar != null) {
+            value = Integer.parseInt(intEnvVar);
+        }
+        return value;
```

Comment:
This version will avoid needing a whole `Map` of all the System properties when you only need one, and also supports a default. I don't think you should rely on the CO setting a value for the env var. 

```java
private static int getIntProperty(String name, int default) {
    String env = System.getEnv(name);
    if (env != null) {
        return Integer.parseInt(env)
    } else {
        return default;
    }
}
```